Makefile

```diff
@@ -91,13 +102,6 @@ vet:
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```

Comment:
I'm not 100% sure, but I think the spirit of what this is doing is similar to what [boilerplate's `op-generate`](https://github.com/openshift/route-monitor-operator/blob/97404eca7e5b669d904607c6a357d459fecb9116/boilerplate/openshift/golang-osd-operator/standard.mk#L117-L124) does -- namely [`operator-sdk generate crds/k8s`](https://github.com/openshift/route-monitor-operator/blob/97404eca7e5b669d904607c6a357d459fecb9116/boilerplate/openshift/golang-osd-operator/operator-sdk-generate.sh#L46-L47) (kubebuilder etc).

As written, this overrides [boilerplate's `generate`](https://github.com/openshift/route-monitor-operator/blob/97404eca7e5b669d904607c6a357d459fecb9116/boilerplate/openshift/golang-osd-operator/standard.mk#L137-L138), which gets used in CI ([`validate`](https://github.com/openshift/route-monitor-operator/blob/97404eca7e5b669d904607c6a357d459fecb9116/boilerplate/openshift/golang-osd-operator/standard.mk#L182-L185) => [`generate-check`](https://github.com/openshift/route-monitor-operator/blob/97404eca7e5b669d904607c6a357d459fecb9116/boilerplate/openshift/golang-osd-operator/standard.mk#L155-L160)). If you could rename yours `op-generate` (and remove `op-generate` from [this override](https://github.com/openshift/route-monitor-operator/pull/82/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R17)), that just might plug it into the intended part of the chain.

Reply:
this is generated by  operator-sdk. I don't think `operator-sdk generate crds/k8s` is a valid command in operator-sdk v1

Reply:
Right, it's not, my point is that we want `make op-generate` to do whatever the v1 equivalent is; and we want `make generate` to be the broader target that includes go and openapi (and any other generators we add in the future).

Reply:
I think all generation commands are provided by the SDK, no reason to use one from boilerplate.